corehq/apps/api/resources/v0_5.py

```diff
@@ -344,6 +344,9 @@ def get_resource_uri(self, bundle_or_obj=None, url_name='api_dispatch_detail'):
     def _update(self, bundle):
         should_save = False
         for key, value in bundle.data.items():
+            if key == "role":
+                # role handled in _set_role_for_bundle
+                continue
```

Comment:
Can you help understand how this was even working before? I see that `obj_create` calls `_set_role_for_bundle` but this method does not. Should that be done now?

Reply:
That's correct. This API does not support updating a web user's role after creation.

Reply:
thanks @snopoke

Reply:
then is this a correct statement? `role handled in _set_role_for_bundle`

Could this be "role updates are not supported, ignore" or something else to make that clear?

Reply:
`_update` is called for both creating a user and updating a user so the role field here is not being ignored, it is just handled outside of this method (but only for the 'create' case). I don't know if the lack of role update for updating a user is an oversight or not but I don't want to change that in this PR.

Reply:
that makes sense. Didn't realize this was not the actual overall "update".